.github/workflows/deploy.yml

```diff
@@ -26,8 +26,9 @@ jobs:
           ref: ${{ matrix.branch }}
       - name: Audit
         run: |
-          cargo audit -D warnings --ignore RUSTSEC-2020-0016
+          cargo audit -D warnings --ignore RUSTSEC-2020-0016 --ignore RUSTSEC-2020-0095
```

Comment:
We should just really replace that crate, `cargo-audit` itself proposes some alternatives:
- [`dissimilar`](https://crates.io/crates/dissimilar)
- [`similar`](https://crates.io/crates/similar)
- [`treediff`](https://crates.io/crates/treediff)
- [`diffus`](https://crates.io/crates/diffus)

I know `similar` was made a direct replacement because of the lack of maintenance.

Reply:
We aren't directly using difference. assert_cmd is using predicates which is using this crate. We'd have to replace assert_cmd, but given the popularity of that crate and the fact that it's still in the rust book and apparently maintained, I would prefer just letting this in since it only runs during unit tests.

Reply:
That sounds fine then, I can do a PR at some point, low prio.